### ISSUE TITLE
No changelog, tags, release notes or releases available

### ISSUE THREAD

**Issue by ristomatti** [LoopFiftyFour/JS-Connector#33]
Currently no information is provided what changes between the versions of this module. There's also no releases to get notified of.  How are developers expected to know when the library updates and to check if the update is relevant to their project? Scan through the commit list every now and then?

It should not be too much overhead to provide the minimum required to fulfill both needs. Just automate the NPM publish workflow using GitHub Actions. There's also interactive CLI tools available to help you do this: https://www.npmjs.com/package/release-it

I would suggest to check it out. It might even simplify your own workflow.

**Comment by joelwkall** [comment]
Good point, we'll discuss and see whether we'll prioritize this. For now I think you don't have to keep track of updates, if we make changes that are relevant to the implementations you're maintaining we will make sure to let you know.

**Comment by micke-dahlgren** [comment]
Do you follow semantic versioning? [https://semver.org/](https://semver.org/)

**Comment by joelwkall** [comment]
> Do you follow semantic versioning? https://semver.org/

Yes, at least that is the intent :) But there may have been instances in the past where we've made mistakes.

**Comment by ristomatti** [comment]
Any update on this?
